tests/codes/process/test_setup.py

```diff
@@ -38,19 +38,18 @@ class TestSetup:
     def setup_method(self):
         self.default_pf = ProcessSolverParams.from_json(PARAM_FILE)
 
-        self._writer_patch = mock.patch(f"{MODULE_REF}.InDat")
-        self.writer_cls_mock = self._writer_patch.start()
-
-    def teardown_method(self):
-        self._writer_patch.stop()
```

Comment:
I think we still need teardowns for the patches, or they're going to persist past the life of the class, and (potentially) mess with subsequent tests.